docs/playbook-reference/actions/scans.rst

```diff
@@ -0,0 +1,48 @@
+Scans
+########
+
+Robusta includes built-in actions to scan and get insights on Kubernetes clusters.
+
+These actions can be triggered automatically on a schedule, manually using the Robusta Platform and :ref:`manually using the robusta cli <Manual Triggers>`.
+
+
+Popeye - A Kubernetes Cluster Sanitizer
+---------------------------------------------
+
+`Popeye <https://github.com/derailed/popeye>`_ is a utility that scans live Kubernetes cluster and reports potential issues with deployed resources and configurations. by default Robusta will run a Popeye scan on startup.
+
+Popeye scans can be triggerd and viewed in the Robusta platform
+
+.. image:: /images/popeye_example.png
+    :width: 1000
+    :align: center
+
+.. robusta-action:: playbooks.robusta_playbooks.popeye.popeye_scan
```

Comment:
Can you update the `.. robusta-action` directive so that it shows an on_schedule trigger? You should be able to add it as a second paremter to control the generated example:

```
.. robusta-action:: playbooks.robusta_playbooks.popeye.popeye_scan on_schedule
```